### PR TITLE
Update spawn and remove plugins

### PR DESCRIPTION
**Pull request by bernardomovai**
Update spawn and removal plugins. Both plugins now receive a string with the names of the models to spawn/remove.
Relates to
 [https://movai.atlassian.net/browse/RP-1727](url) 
and
[https://movai.atlassian.net/browse/RP-1728](url)

- [x] Make sure you are opening from a **topic/feature/bugfix branch**
- [x] Ensure that the PR title represents the desired changes
- [x] Ensure that the PR description detail the desired changes
- [x] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: spawn_model/SpawnModel.cc [hunk 1]
```diff
@@ -38,15 +38,20 @@ void SpawnModel::Configure(const Entity &_entity,
...
 
-  // Get the components name to create the spawn topic subscriber
+  // Get the component's name to create the spawn topic subscriber
   std::string objectName = _ecm.Component<components::Name>(_entity)->Data();
   this->worldName = _ecm.Component<components::Name>(worldEntity)->Data();
-  //Change name of the topic to subscribe to
+  //Create topics to subscribe to 
+  // - "create" -> PoseArray message to get the pose of each object to spawn
+  // - "model_array" -> name of the models to spawn
   std::string topic = "/world/" + worldName + "/create";
```
Comment (filipa-movai) [inline spawn_model/SpawnModel.cc:47]: these are not very good variable names

**Review by filipa-movai — APPROVED** [review]

**Comment by filipa-movai** [comment]
Looks good to me but would like to get @jmspribeiro opinion since it is more his area of expertise

File: spawn_model/SpawnModel.cc [hunk 2]
```diff
@@ -64,63 +69,85 @@ void SpawnModel::Update(const ignition::gazebo::UpdateInfo &_info,
...
-      // Create the spawn service topic using the message informations and call the service to spawn a model
-      const std::string srvCreate = "/world/" + this->worldName + "/create";
-      //  ignwarn << "Created " << reqStr << std::endl;
-      this->node.Request(srvCreate, req,timeout,res,result);
-      ignmsg << "Created Object" << std::endl;
-    }
-    this->spawnObject = false;
-    
+      for(int i = 0; i<this->objectPosition.size();i++){
+        std::string reqStr = std::string("<sdf version='1.7'>") +
```
Comment (jmspribeiro) [inline spawn_model/SpawnModel.cc:82]: Consider using a xml library for writing xml content (tinyxml is a good one)

**Review by jmspribeiro — APPROVED** [review]
LGTM

**Review by jmspribeiro — CHANGES_REQUESTED** [review]
please bump the version

**Review by jmspribeiro — APPROVED** [review]

**Pull request merged**